src/PhpWord/Settings.php

```diff
@@ -29,17 +29,17 @@ class Settings
      *
      * @const string
      */
-    const ZIPARCHIVE = 'ZipArchive';
-    const PCLZIP = 'PclZip';
-    const OLD_LIB = 'PhpOffice\\PhpWord\\Shared\\ZipArchive'; // @deprecated 0.11
+    const ZIPARCHIVE = \ZipArchive::class;
```

Comment:
Class `\ZipArchive` is part of `ext-zip` extension. In production install PHPWord does not require `ext-zip`. Class `\ZipArchive` may not exist.